docs/source/installation.md

```diff
@@ -1,51 +1,95 @@
-
 # Installation
 
-## Installation using conda
+xcube can be installed from a released conda package, or directly from a
+copy of the source code repository.
+
+The first two sections below give instructions for installation using conda,
+available as part of the [miniconda
+distribution](https://docs.conda.io/en/latest/miniconda.html). If installation
+using conda proves to be unacceptably slow, mamba can be used instead (see
+[Installation using mamba](#installation-using-mamba)).
+
+## Installation from the conda package
 
-Into existing conda environment (>= Python 3.7)
+Into a currently active, existing conda environment (>= Python 3.7)
 
     $ conda install -c conda-forge xcube
 
-Into new conda environment
+Into a new conda environment named `xcube`:
     
-    $ conda create -c conda-forge -n xcube python3
-    $ conda install -c conda-forge xcube
+    $ conda create -c conda-forge -n xcube xcube
 
+The argument to the `-n` option can be changed to create a differently
+named environment.
 
-## Installation from sources
+## Installation from the source code repository
 
-First
+First, clone the repository and create a conda environment from it:
     
     $ git clone https://github.com/dcs4cop/xcube.git
     $ cd xcube
     $ conda env create
     
-Then
+The `conda env create` command above creates an environment according to
+the specifications in the `environment.yml` file in the repository, which
+by default takes the name `xcube`. Then, to activate the environment and
+install xcube from the repository:
     
     $ activate xcube
-    $ python setup.py develop
+    $ pip install --no-deps --editable .
+
+The second command installs xcube in ‘editable mode’, meaning that it will
+be run directly from the repository, and changes to the code in the repository
+will take immediate effect without reinstallation. (As an alternative to
+pip, the command `python setup.py develop` can be used, but this is
+[no longer recommended](https://docs.python.org/3/install/#introduction).
+Among other things, `pip` has the advantage of allowing easy deinstallation of
+installed packages.)
 
-Update
+To update the install to the latest repository version:
     
     $ activate xcube
     $ git pull --force
-    $ python setup.py develop
     
```

Comment:
```suggestion
    If the packages in environment.yml have changed, please update your conda environment with: 
        $ cd xcube
        $ conda env update -n xcube --file environment.yml --prune    
```

Reply:
> If the packages in environment.yml have changed, please update your conda environment with: 

A user cannot know this easily!

May be better to say

"From time to time xcube's pacjage dependencis may change. If you recieve error messages about missing modules, , please update your conda environment with:"

Reply:
I'm going to add the update command to the update instructions right after the "git pull" and without any conditional, just an explanatory note that it's sometimes necessary. In most cases it will be a no-op, but executing it is easier than checking whether one needs to execute it :).